cmake/arm64-windows-llvm.cmake

```diff
@@ -0,0 +1,16 @@
+set( CMAKE_SYSTEM_NAME Windows )
+set( CMAKE_SYSTEM_PROCESSOR arm64 )
+
+set( target arm64-pc-windows-msvc )
+
+set( CMAKE_C_COMPILER    clang )
+set( CMAKE_CXX_COMPILER  clang++ )
+
+set( CMAKE_C_COMPILER_TARGET   ${target} )
+set( CMAKE_CXX_COMPILER_TARGET ${target} )
+
+set( arch_c_flags "-march=armv8.7-a -fvectorize -ffp-model=fast" )
```

Comment:
Isn't this a bit too much? 
We need to show some love for the 8cx series 😉, aren't they only 8.2-a (or maybe 8.3-a)?

Reply:
Yeah, armv8.2-a  should be enough in general. I'll double check the specs and update accordingly. 
And yes, to answer your previous question this update helps 8cx as well.
Not sure about QNN support though.

Reply:
Ok. I double checked the specs. 8cx is armv8.4-a. 
Unfortunately, that doesn't include MatMul instructions (those were introduced in v8.6-a). 
Without the matmul the token rate on X-Elite drops down to 40/19 t/s. 
So, for the newer (last couple of years) Snapdragon platforms (8-Gen 2/3,  X-Elite, ...) we need at least armv8-6a to enable matmul.
If folks are interested in supporting 8cx perhaps we should add explicit cmake target/toolchain for that.